src/pages/yujinLee/Login/Login.js

```diff
@@ -1,6 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+// import { Link } from "react-router-dom";
+import './Login.scss';
+
 const Login = () => {
-  return <h1>Login입니다.</h1>;
+  const navigate = useNavigate();
+
+  const [userId, setUserId] = useState('');
+  const [userPw, setUserPw] = useState('');
+
+  const saveUserId = e => {
+    setUserId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setUserPw(e.target.value);
+  };
```

Comment:
계산된 속성명 활용!
input이 여러개가 되면 state가 여러개가 되기 때문에 유지보수, 확장성 측면에서 계산된 속성명을 활용하는 게 좋다!
```suggestion
  const [userInput, setUserInput] = useState({
    id: "",
    pw: "",
  });

  const handleUserInput = (e) => {
    setUserInput({ ...userInput, [e.target.name]: e.target.value });
  };
```